src/runtime/data/kata-collect-data.sh.in

```diff
@@ -467,6 +467,22 @@ show_container_mgr_details()
 		subheading "$title"
 		run_cmd_and_show_quoted_output "" "podman --version"
 
+		run_cmd_and_show_quoted_output "" "podman system info"
+
+		local cmd file config_files
+		config_files+="/usr/share/containers/containers.conf"
+		config_files+=" /etc/containers/containers.conf"
+		config_files+=" /usr/share/containers/mounts.conf"
+		config_files+=" /etc/containers/policy.json"
+		config_files+=" /etc/containers/registries.conf"
+		config_files+=" /etc/containers/storage.conf"
+		for file in $config_files; do
```

Comment:
Couldn't you go with a large hammer and do something like:

```
for file in {/etc,/usr/share}/containers/*.conf ; do ...
```

If there is any bizarre new conf file showing up in a future version, we will get it that way.

Reply:
Smart!
Hmm, the only thing here is that the /etc/containers folder serves multiple projects: podman, buildah, skopeo and cri-o.
While I expect that files landing there will be files used by all the projects, we don't know what will happen... True is that in the worst case (let's say that at certain point a new file will be dropped there just for skopeo and buildah) we will just display an unrelated file, no harm at all.
Don't know... here I would be *slightly* in favour to keep the files displayed under strict control. @c3d WDYT?

Reply:
Worst case is that we would get config files we don't really need. Or maybe in some subtle bug it would actually matter that we have these. If it becomes too chatty, we'll advise. Code is fluid.

Reply:
Or if you really want to restrict, then maybe use a pattern like `{/etc,/usr/share}/containers/{policy.json,registries.conf,storage.conf}`. That would still make the code more readable and easier to extend later IMO.

Reply:
ok, let's dump all the files then.
Pushed the updated version (and rebased)